cmd/vmproxy/main.go

```diff
@@ -36,13 +33,13 @@ func main() {
 	var ln net.Listener
 
 	ln = must.Get(s.Listen("tcp", ":22"))
-	go sshServer(ln)
+	go vmproxy.SshServer(ln, vmName)
```

Comment:
Capitalization of common initialisms

```suggestion
	go vmproxy.SSHServer(ln, vmName)
```